<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-70RPUI0" failures="1" tests="3" name="Person" time="14.958" errors="0" timestamp="2022-08-21T10:07:41 IST">
  <testcase classname="com.qa.testCases.TC_PersonMRV" name="validatePersonHeader" time="5.478">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.testCases.TC_PersonMRV.validatePersonHeader(TC_PersonMRV.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validatePersonHeader -->
  <testcase classname="com.qa.testCases.TC_PersonSRV" name="validateSRVPerson" time="8.758"/>
  <testcase classname="com.qa.testCases.TC_PersonArchive" name="validateArchive" time="0.0">
    <skipped/>
  </testcase> <!-- validateArchive -->
  <testcase classname="com.qa.base.BaseClass" name="@BeforeSuite setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setup -->
  <testcase classname="com.qa.base.BaseClass" name="@BeforeSuite setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setup -->
  <testcase classname="com.qa.base.BaseClass" name="@AfterSuite tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase classname="com.qa.base.BaseClass" name="@AfterSuite tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
</testsuite> <!-- Person -->
